contracts/extensions/OldRolesFactory.sol

```diff
@@ -25,12 +25,12 @@ import "./ExtensionFactory.sol";
 import "./OldRoles.sol";
 
 
-contract OldRolesFactory is ExtensionFactory, ColonyDataTypes {
+contract OldRolesFactory is ExtensionFactory, ColonyDataTypes { // ignore-swc-123. TODO Any takers to explain to me?
   mapping (address => OldRoles) public deployedExtensions;	
 
   function deployExtension(address _colony) external {
     require(
-      ColonyAuthority(IColony(_colony).authority()).hasUserRole(msg.sender, 1, uint8(ColonyRole.Root)) == true, 
+      ColonyAuthority(IColony(_colony).authority()).hasUserRole(msg.sender, 1, uint8(ColonyRole.Root)) == true, // ignore-swc-123
```

Comment:
If we change this to `IColony(_colony).hasUserRole(msg.sender, 1, ColonyRole.Root)` these errors go away. Same for the other factory.